spec/rubocop/cop/rspec/stubbed_mock_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::StubbedMock do
+  subject(:cop) { described_class.new }
+
+  it 'flags stubbed message expectation' do
+    expect_offense(<<-RUBY)
+      expect(foo).to receive(:bar).and_return('hello world')
+      ^^^^^^^^^^^ Prefer `allow` to `expect` when configuring a response.
+    RUBY
+  end
+
+  it 'flags stubbed message expectation with a block' do
+    expect_offense(<<-RUBY)
+      expect(foo).to receive(:bar) { 'hello world' }
+      ^^^^^^^^^^^ Prefer `allow` to `expect` when configuring a response.
+    RUBY
+  end
+
+  it 'flags stubbed message expectation with argument matching' do
+    expect_offense(<<-RUBY)
+      expect(foo).to receive(:bar).with(42).and_return('hello world')
+      ^^^^^^^^^^^ Prefer `allow` to `expect` when configuring a response.
+    RUBY
+  end
+
+  it 'flags stubbed message expectation with argument matching and a block' do
+    expect_offense(<<-RUBY)
+      expect(foo).to receive(:bar).with(42) { 'hello world' }
+      ^^^^^^^^^^^ Prefer `allow` to `expect` when configuring a response.
+    RUBY
+  end
+
+  it 'ignores `have_received`' do
+    expect_no_offenses('expect(foo).to have_received(:bar)')
```

Comment:
Just for consistency (making the spec file easier to read), could you please use a heredoc in this example too?